test/Microsoft.Extensions.ProjectModel.Tests/DotNet/DotNetDependencyProviderTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.ProjectModel;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Extensions.ProjectModel.DotNet
+{
+    public class DotNetDependencyProviderTests
+    {
+        private const string NugetConfigTxt = @"
+<configuration>
+    <packageSources>
+        <clear />
+        <add key=""NuGet"" value=""https://api.nuget.org/v3/index.json"" />
+        <add key=""dotnet-core"" value=""https://dotnet.myget.org/F/dotnet-core/api/v3/index.json"" />
+        <add key=""dotnet-buildtools"" value=""https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json"" />
+        <add key=""nugetbuild"" value=""https://www.myget.org/F/nugetbuild/api/v3/index.json"" />
+    </packageSources>
+</configuration>";
+
+        private const string projectJson = @"
+{
+  ""buildOptions"": {
+  },
+  ""dependencies"": {
+    ""Microsoft.AspNetCore.Mvc"": ""1.0.0-*"",
```

Comment:
This is test for Abstraction using the Project.json. 
The MsBuild related tests are in the MsBuildDependencyProviderTests.